Activity 02

Droxinbox opened the pull request:
The Activity is now completed and ready for review.

Droxinbox committed:
Activity 02

The Activity is now completed and ready for review.

erobledoapex commented:
Las instrucciones también piden una captura de pantalla para cada entrega

erobledoapex commented on ExercisePart1/src/app/app.component.html:
```diff
@@ -1,336 +1,54 @@
+
+    <div class="mt-3 mb-3">
+      <ul class="list-group" style="max-width: 400px; border: 2px solid; margin: 10px;">
+        <li class="list-group-item" *ngFor="let item of test; index as i" [ngSwitch]="item">
```
aquí no se usa el index, solo hay que declarar variables si van a ser usadas

erobledoapex commented on ExercisePart1/src/app/app.component.html:
```diff
@@ -1,336 +1,54 @@
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
 
+</main>
```
favor de crear un componente por cada ejercicio

Droxinbox committed:
Activity 02 ver2.0

+ Activity 02 has been converted into a component.
+ Unnecesary lines of code has been removed.

Droxinbox commented:
![imagen](https://github.com/Droxinbox/AngularPractices/assets/36482757/c4adc29a-10c7-4db0-a633-74a30644dc27)

Droxinbox commented:
![imagen](https://github.com/Droxinbox/AngularPractices/assets/36482757/c7654131-3f4c-4012-a263-b7207b158bcf)

Droxinbox committed:
Merge branch 'main' into Activity_02
